cwms-data-api/src/test/java/cwms/cda/api/TimeSeriesRecentControllerIT.java

```diff
@@ -0,0 +1,255 @@
+ * in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package cwms.cda.api;
+
+import static cwms.cda.api.Controllers.*;
+import static cwms.cda.data.dao.DaoTest.getDslContext;
+import static cwms.cda.security.KeyAccessManager.AUTH_HEADER;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import cwms.cda.api.enums.VersionType;
+import cwms.cda.api.errors.NotFoundException;
+import cwms.cda.data.dao.TimeSeriesCategoryDao;
+import cwms.cda.data.dao.TimeSeriesDaoImpl;
+import cwms.cda.data.dao.TimeSeriesGroupDao;
+import cwms.cda.data.dto.AssignedTimeSeries;
+import cwms.cda.data.dto.TimeSeries;
+import cwms.cda.data.dto.TimeSeriesCategory;
+import cwms.cda.data.dto.TimeSeriesGroup;
+import cwms.cda.formatters.ContentType;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.json.JsonV1;
+import fixtures.CwmsDataApiSetupCallback;
+import fixtures.TestAccounts;
+import io.restassured.filter.log.LogDetail;
+import javax.servlet.http.HttpServletResponse;
+import java.sql.Timestamp;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import mil.army.usace.hec.test.database.CwmsDatabaseContainer;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.DSLContext;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+
+@Tag("integration")
+class TimeSeriesRecentControllerIT extends DataApiTestIT {
+    TestAccounts.KeyUser user = TestAccounts.KeyUser.SPK_NORMAL;
+    private static final Logger LOGGER = Logger.getLogger(TimeSeriesRecentControllerIT.class.getName());
+    private static final String OFFICE_ID = "SPK";
+    private static final String LOCATION = "ZACK1";
+    private static final String TS_ID = LOCATION + ".Depth.Inst.15Minutes.0.TS_TEST";
+    private static final String CATEGORY_ID = "TEST";
```

Comment:
The location name itself is fine, please use a more "real" name for the category and group.

Reply:
Replaced with more realistic naming. 

Also fixed the integration test, had to swap from hard-coded dates to using now() for ensuring TS dates are within recent time window